### ISSUE TITLE
Firefox autofill pops up on non-login fields, blocks actual website UI

### ISSUE THREAD

**Issue by ross-w** [strongbox-password-safe/Strongbox#704]
The Strongbox autofill extension for Firefox appears on many different fields across a wide variety of websites. This is great for logins, but in many cases the popup is unwarranted.

Worse still, many times this popup appears and obscures the drop down fields provided by the actual website. There's no way to dismiss the Strongbox popup and use the popup underneath, so this makes the dropdown unusable.

Example 1: Address input field which has an autofill from Google. The answer I want to give is below the popup:
![example1](https://github.com/strongbox-password-safe/Strongbox/assets/2508364/f822bf93-bff0-4bf3-9a92-cbd6546633fb)

Example 2: Xero (accounting software) when selecting an account. Autofill is here for all the accounts to choose from, but it is blocked by the popup
![example2](https://github.com/strongbox-password-safe/Strongbox/assets/2508364/aee48451-cba5-438e-bd53-86cfb28743b5)

I'm not sure how feasible it is to stop this appearing everywhere, but at least an option to hide the popup or only show it when a key combination is pressed would be very useful

**Comment by strongbox-mark** [comment]
Hi @ross-w - There's an option to disable inline menus per domain on the Settings tab for the AutoFill extension. Have you tried that?

**Comment by ross-w** [comment]
@strongbox-mark wow, ok. I didn't realise there was even a button (Firefox had buried it under the extensions button) for settings. Thanks, will give this a go. I assume there's a way to have autofill work a different way if I disable the in-line menus?

**Comment by strongbox-mark** [comment]
Yes, for sure, you can click on the button (once you've made it visible) and you'll see the suggestions there. You can click on these to fill, or use the Cmd+Shift+L keyboard shortcut to fill.

**Comment by ross-w** [comment]
Thanks @strongbox-mark that will do for now. I don't know if you want to close this one or still leave it open as it pops up on non-login fields? FWIW I switched from 1password and that did not have the same issue. But it might be (just guessing) a case of them constantly maintaining a database of field names or something which may be too much overhead for the developers here.

**Comment by strongbox-mark** [comment]
Yes, it's a difficult one to fine tune actually, and the idea of maintaining a list somewhere doesn't appeal. It might be better to give the user an option to choose how aggressive Strongbox is in determining if a field is login... e.g. in your example, I assume the Xero field has an "Account" attribute which in a lot of cases is actually a login field. 

Having an option to disable should help most people, but yeah could definitely do with a little more granular control. Happy to leave open for consideration.

**Comment by sammcj** [comment]
I had this issue as well, I tried out disabling the inline menus but most of the time I like them. I wonder if an option would be to add a <...> to fields that when clicked brings up the inline menu?

Also I see your comment about the keyboard shortcut Mark - I just happened to log an issue for this in the extension repo! https://github.com/strongbox-password-safe/browser-autofill/issues/4 - cmd+shift+L doesn't seem to work for me - that "loves" the currently playing track in Apple Music.
